### ISSUE TITLE
Sudden FPS drop

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#274]
Originally reported on Google Code with ID 275

```
Describe your system:
- Linux distribution:
- Machine type: (32-bit or 64-bit)
 - Mupen64Plus version: 1.5
 - Plugins used: Rice, JttL, blight, Hacktarux/Azimer

Describe the problem:
Super Mario 64 starts out smoothly, but uses all of my CPU. After a few
seconds the CPU usage drops to ~15%, but the FPS also drop to 1-2, along
with the sound, that starts to sounds jittery.
This happens even if I use the No Sound plugin, which leads me to believe
it is not a sound issue.
If I stop the game and start it again, it will still have extremely low
FPS. The only way to fix it is to restart mupen (but then it happens again
after a few seconds).

Please provide any additional information below.

WORKAROUND:
Using Glide64 Mupen64Plus graphics plugin. Apparently the problem is in
Rice's plugin.
```

Reported by `andrejx` on 2009-09-28 11:43:59

**Comment by richard42** [comment]
```
Did you use a build with DBG (debugger) enabled or the debian/ubuntu build (both have
debugger enabled). This is a problem of the integrated debugger feature of rice which
is (for unknown reason) enabled when the gui debugger is enabled. This is fixed by
http://git.debian.org/?p=collab-maint/mupen64plus.git;a=blob;f=debian/patches/rice_nodebug.patch;h=180813d67c219c8227b76d1ef36730730a27f54e;hb=master
```

Reported by `sven@narfation.org` on 2009-09-28 11:59:13

**Comment by richard42** [comment]
```
I used this one:
http://packages.debian.org/squeeze/mupen64plus

So yes, you are probably right about the debugger causing this.
```

Reported by `andrejx` on 2009-09-28 12:15:23

**Comment by richard42** [comment]
```
I will release a new version soon (without banjo tooie and xz support because I am
waiting for some reactions from other people) which should fix that problem in
Debian. Maybe someone can request a Sync to ubuntu for 1.5+dfsg-5 afterward.
Please use glide64 in the meantime :)
```

Reported by `sven@narfation.org` on 2009-09-28 13:21:16

**Comment by richard42** [comment]
```
Please test it now against the version 1.5+dfsg1-5 in Debian sid/unstable
```

Reported by `sven@narfation.org` on 2009-09-29 08:16:12

**Comment by richard42** [comment]
```
Tested, and it works perfectly!
Congratulations! :D
```

Reported by `andrejx` on 2009-09-29 08:59:11

**Comment by richard42** [comment]
```
Sorry that I missed it when I released 1.5+dfsg1-4... but I am mostly a glide64 user
and noticed it first while debugging bug 51. So my part is done and upstream should
think about what they are doing with it.
```

Reported by `sven@narfation.org` on 2009-09-29 09:43:05

**Comment by richard42** [comment]
```
committed rev 1415
```

Reported by `richard42g` on 2009-10-02 03:08:02
- **Status changed**: `Fixed`
